packages/grant-explorer/src/features/round/ViewProjectDetails.tsx

```diff
@@ -109,6 +125,12 @@ function ProjectTitle(props: { projectMetadata: ProjectMetadata }) {
 }
 
 function AboutProject(props: { projectToRender: Project }) {
+  const [verifiedProviders, setVerifiedProviders] = useState<{
+    [key: string]: VerifiedCredentialState;
+  }>({
+    github: VerifiedCredentialState.PENDING,
+    twitter: VerifiedCredentialState.PENDING,
+  });
```

Comment:
@thelostone-mc are these the only verifiable credentials for this page?

there's personal and company github, i'm not sure about the keys for these. is this something i'll have to ask grant hub folks?

Reply:
Yup it looks like that!
@gravityblast Could you offer some guidance here ?

Reply:
@chibie @thelostone-mc yes we have VC only for project's twitter and github

Reply:
@gravityblast is it personal or project github?

Reply:
@chibie it's verifying that your github account is part of the org you specified